### ISSUE TITLE
Missing visual label for the Search field

### ISSUE THREAD

**Issue by davidstenger** [uswds/uswds#1647]
**Issue**:
The Search Small form element does not have visual form label. See attachment FormLabel_ExtendedHeader.png Same issue is presented for the Close Button in Responsive view. See attachment [ButtonName_ResponsiveViewl_ExtendedHeader.png](https://drive.google.com/open?id=0ByQeU_iOqP_iNFJ6S1JPY2JWZzg)

**Solution**:
All form controls on the page should have a visible label that is programmatically associated using the “label” element that references the form control’s "id" attribute value with the “for” attribute. When a visible label isn’t available or desirable, then it is acceptable to use a place holder text as Search Small inside the field, and use the title attribute to provide an accessible name. For example: < input type="text" title="First name" …> For the Close button in Responsive view, add the visual "Close" name.

**Comment by maya** [comment]
The small search does not have a visual label but it has a label that's accessible to screen readers instead:
`<label class="usa-sr-only" for="search-field-small">Search small</label>`

It seems to have the correct label and output "Search small" via ANDI:
<img width="943" alt="screen shot 2018-02-12 at 4 09 52 pm" src="https://user-images.githubusercontent.com/5249443/36126852-4b6575d0-100f-11e8-8eeb-6f04a251f12e.png">

From w3.org:
>In some cases, the purpose may be clear enough from the context when the content is rendered visually. The label can be hidden visually, though it still needs to be provided within the code to support other forms of presentation and interaction, such as for screen reader and speech input users.

https://www.w3.org/WAI/tutorials/forms/labels/
